tests/Endpoints/ClientTest.php

```diff
@@ -210,28 +210,48 @@ public function testVersion(): void
 
     public function testStats(): void
     {
+        $index = $this->createEmptyIndex($this->safeIndexName('stats'));
+        $index->addDocuments([
+            ['objectID' => 1, 'type' => 'Library'],
+            ['objectID' => 2],
+        ])->wait();
+
         $response = $this->client->stats();
 
-        self::assertArrayHasKey('databaseSize', $response);
-        self::assertArrayHasKey('lastUpdate', $response);
-        self::assertArrayHasKey('indexes', $response);
+        self::assertGreaterThanOrEqual(0, $response->getDatabaseSize());
+        self::assertGreaterThanOrEqual(0, $response->getUsedDatabaseSize());
+
+        $statsIndex = null;
+        foreach ($response->getIndexes() as $indexName => $indexStats) {
+            if (str_starts_with($indexName, 'stats_')) {
+                $statsIndex = $indexStats;
+                break;
+            }
+        }
```

Comment:
I'm under the impression I'm missing something. Why not use the following syntax?

```php
$statsIndex = $response->getIndexes()[$index->getUid()]; 
```